src/WidgetBase.ts

```diff
@@ -706,7 +706,7 @@ export class WidgetBase<P = WidgetProperties, C extends DNode = DNode> extends E
 			let child: WidgetBaseInterface<WidgetProperties>;
 
 			if (!isWidgetBaseConstructor(widgetConstructor)) {
-				const item = this.getRegistries().get(widgetConstructor);
+				const item = this.registries.get(widgetConstructor);
```

Comment:
is this the only place the `registries` are accessed? just noticed that we've renamed it back now.

Reply:
I think everywhere else actually accesses the private (`_registries`) in `WidgetBase`, I'll update this to do that.